Servers/UI/OJS.Servers.Ui/ClientApp/src/common/app-url-types.tsx

```diff
@@ -3,8 +3,8 @@
 import { ContestParticipationType } from './constants';
 
 interface IRegisterForContestTypeUrlParams {
-    id: number;
-    participationType: ContestParticipationType;
+    id?: number;
```

Comment:
Why is this change needed?

Reply:
So that I avoid a cast in SubmissionDetails when getting the registerContestTypeUrl, but I revert it and just cast it now.